static/app/views/projectInstall/createProject.tsx

```diff
@@ -405,11 +496,15 @@ function CreateProject() {
 
           {errors && (
             <Alert type="error">
-              {Object.keys(errors).map(key => (
-                <div key={key}>
-                  <strong>{startCase(key)}</strong>: {errors[key]}
-                </div>
-              ))}
+              {Object.keys(errors).map(key => {
+                const label =
+                  key === 'actions' ? 'Notify via integration' : startCase(key);
```

Comment:
```suggestion
                  key === 'actions' ? t('Notify via integration') : startCase(key);
```

Also you probably don't want to just take the key and capitalize the first letter, instead what you probably want is a mapping of keys to translated strings


```tsx
const errors: Record<typeof key, string> = {
  key1: t('Key1'),
  key2: t('Key2'),
  ...
}
```

The `Record<typeof key, string>` is just an example, if the keys aren't actually know at type check time it could just be `Record<string, string>`

Something important to remmeber is that you can't give `t()` dynamic values like `t(startCase(key))`. This is because the way translations are extracted and sent to our translators is that we statically analyze our code and extract out the strings inside of the `t("string")` calls. So if you did `t(dynamicVar)` the extractor doesn't know what the string is and can't include it in the list of strings to translate.

Reply:
I'm assuming they used `startCase` to be able to generically handle any errors that could occur- I'm not familiar enough with all of the possible errors to be able to make a record here 😕 I just wanted to handle this special "actions" case because this label is not clear where the error occurred (my guess is that this error was originally intended to only be shown on the Alert Creation page, where it's more clear)